instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -44,16 +43,6 @@
   private static final SupportabilityMetrics supportability =
       new SupportabilityMetrics(Config.get()).start();
 
-  // Keeps track of the server span for the current trace.
-  // TODO(anuraaga): Should probably be renamed to local root key since it could be a consumer span
-  // or other non-server root.
```

Comment:
I've removed this TODO because it's possible to have a CONSUMER->CONSUMER trace (in case of [batch receiving](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#batch-receiving)).